### DISCUSSION TITLE
Let kepler pod fail when no access to eBPF

### DISCUSSION THREAD

**Discussion by jochen-schuettler** [sustainable-computing-io/kepler#1147] [Ideas]
I had a strange problem that kepler monitored only 0 on a given machine. The log showed that accessing eBPF failed. Kepler 0.7.x solved the issues, likely the "libbpf as standard" helped. But I think I would have found the problem much quicker if the kepler pod would fail into an "ERROR" state in that case, instead of going into normal "running" state.

**Comment by marceloamaral** [comment]
@jochen-schuettler good point. 

Given that the current version is entirely dependent on eBPF metrics, we should trigger a Kepler failure if eBPF is not functioning correctly.
